src/directives/auth-login-form-directive.js

```diff
@@ -0,0 +1,27 @@
+(function (angular) {
+    'use strict';
+    var module = angular.module('angular-token-auth.auth-login-form', [
+        'angular-token-auth.auth-login-form-directive-factory',
+    ]);
```

Comment:
I think this file would probably be best combined with something else, in a file that represents the context this is in. I think separating it makes it confusing to understand what it's meant to be doing.